rbc/externals/utils.py

```diff
@@ -0,0 +1,25 @@
+from rbc.targetinfo import TargetInfo
```

Comment:
These functions could also live in `rbc/externals/__init__.py` or in `rbc/externals/irtools.py` or here? Your's choice.
Just saying because in the above I suggested moving `register` functions to `__init__.py`.